app/src/main/java/de/tum/in/tumcampusapp/component/ui/transportation/api/MvvClient.kt

```diff
@@ -0,0 +1,37 @@
+package de.tum.`in`.tumcampusapp.component.ui.transportation.api
+
+import android.content.Context
+import com.google.gson.GsonBuilder
+import de.tum.`in`.tumcampusapp.api.app.Helper
+import org.joda.time.DateTime
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+class MvvClient {
+    companion object {
+        private const val BASE_URL = "http://efa.mvv-muenchen.de/mobile/" // No HTTPS support :(
```

Comment:
Are we sure, that there is no https? It just seemed to work a minute ago, but now I'm getting an error message.

Reply:
Ok, seems like they have HTTPS now, changeing that